xml/docu_styleguide.audience.xml

```diff
@@ -34,7 +34,8 @@
  </para>
 
  <para>
-  Generally, there is no need to add information about the target audience
-  to the book or article content itself.
+  Information about the target audience does not need to be included in the book
+  or article content itself. If you decide to include the target audience,
+  add it to the book preface (or abstract of the article).
```

Comment:
It would make most sense to me to discuss among the team if we want to add to/amend this rule that is widely ignored anyway. At this point, to me, it would make most sense to remove this entire rule from the style guide.

But as an immediate idea, I'd suggest just keeping your first sentence, especially in light of Dima's preface-to-appendix experiment:

```suggestion
  Information about the target audience does not need to be included in the book
  or article content itself.
```

Reply:
As this does not imply any change in meaning, I took the liberty of committing this proposal directly with https://github.com/SUSE/doc-styleguide/commit/9abd378378345bd9e654754900cae5f6fb799cc7.